person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age:, name: 'Unknown', parent_permission: true)
+    @id = nil
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def can_use_services?
+    is_of_age? || @parent_permission
+  end
+
+  private
+
+  def of_age?
+    age >= 18
+  end
```

Comment:
- [ ] You named the method on line 18 `of_age` but called `is_of_age` on line 13. Kindly make the necessary changes.